Adds index files to the new Payment Services Extension folder and subfolders in GraphQL docs (#390)
AdobeDocs/commerce-webapi: src/pages/graphql/payment-services-extension/queries/index.md

---
title: Payment Services extension queries
description: Learn how to use the GraphQL queries to integrate the Payment Services extension.
keywords:
  - GraphQL
  - Payments
---

# Payment Services extension queries

> The Payment Services extension queries enable you to retrieve payment-related data, such as transaction details, payment statuses, and more, allowing you to efficiently manage and monitor your payment services extension.

Reviewer: ```suggestion
The queries described in this section allow you to retrieve payment-related data, such as transaction details and payment statuses so that you can manage and monitor the operations of the Payment Services extension.
```
